js/menu.js

```diff
@@ -9,4 +9,27 @@ function showMenu() {
 function hideMenu() {
   document.querySelector('.navigation.active').classList.remove('active');
   document.querySelector('.menu-list.active').classList.remove('active');
-}
\ No newline at end of file
+}
+
+// -----------Validate Form--------
+
+
+
+document.getElementById("in-touch").onclick = function() {
+  const email = document.querySelector('#e-mail');
+  const popForm = document.querySelector('.formPopup');
+  const submit = document.getElementById('in-touch');
+
+    if (email.value.toLowerCase() !== email.value) {
+      popForm.style.display = 'flex';
+      
+    } else {
+       popForm.style.display = 'none';
+    }
+  }
```

Comment:
- [ ]  You should validate your inputs, at least that the email should be always in lowercase.  When an error occurs, it should appear below the submit button, as the requirement suggested. 
- [ ] Use DOM to Display the error message, and disable the submission in case, there is an error